test/system/shoot_cp_migration/migrate_test.go

```diff
@@ -171,6 +173,11 @@ func beforeMigration(ctx context.Context, t *ShootMigrationTest, guestBookApp *a
 		return err
 	}
 
+	t.GardenerFramework.Logger.Infof("Shoot.Status.IsHibernated = %t", t.Shoot.Status.IsHibernated)
+	if t.Shoot.Status.IsHibernated {
```

Comment:
Nit: 
The log does not harm but I think it is only needed if the shoot is hibernated. 
Otherwise the next `ginkgo.By("Creating test Secret and Service Account")` will be logged anyway.
```suggestion
	if t.Shoot.Status.IsHibernated {
	  t.GardenerFramework.Logger.Infof("Shoot is hibernated no further tests are executed")
```